Client/mods/deathmatch/logic/luadefs/CLuaPedDefs.cpp

```diff
@@ -1028,6 +1036,15 @@ std::variant<bool, CLuaMultiReturn<float, float, float>> CLuaPedDefs::GetElement
     return false;
 }
 
+std::variant<bool, CLuaMultiReturn<float, float, float, float>> CLuaPedDefs::GetElementBoneRotationQuaternion(lua_State* const luaVM, CClientPed* entity, std::int32_t boneId)
+{
+    float    x = 0.0f, y = 0.0f, z = 0.0f, w = 0.0f;
+    CEntity* theEntity = entity->GetGameEntity();
+    if (theEntity && theEntity->GetBoneRotationQuat(static_cast<eBone>(boneId), x, y, z, w))
```

Comment:
`boneId` have to be checked whether it match **eBone** min and max value or not. I would also replace `int` to `uint`.

Reply:
below